src/python/turicreate/toolkits/drawing_classifier/drawing_classifier.py

```diff
@@ -1180,3 +1185,37 @@ def evaluate(self, dataset, metric='auto'):
           >>> print results['accuracy']
         """
         return self.__proxy__.evaluate(dataset, metric)
+
+    def _get_summary_struct(self):
+        """
+        Returns a structured description of the model, including (where
+        relevant) the schema of the training data, description of the training
+        data, training statistics, and model hyperparameters.
+
+        Returns
+        -------
+        sections : list (of list of tuples)
+            A list of summary sections.
+              Each section is a list.
+                Each item in a section list is a tuple of the form:
+                  ('<label>','<field>')
+        section_titles: list
+            A list of section titles.
+              The order matches that of the 'sections' object.
+        """
+        model_fields = [
+            ('Number of classes', 'num_classes'),
+            ('Feature column', 'feature'),
+            ('Target column', 'target')
+        ]
+        training_fields = [
+            ('Training Iterations', 'max_iterations'),
+            ('Training Accuracy', 'training_accuracy'),
+            ('Validation Accuracy', 'validation_accuracy'),
+            ('Training Time', 'training_time'),
+            ('Number of Examples', 'num_examples'),
+            ('Final Loss (specific to model)', 'training_log_loss')
```

Comment:
On further thought, for backward compatibility reasons, I don't actually think we should change the name of the state variable from `training_loss` to `training_log_loss`. @nickjong @znation @srikris  What do you think?

Reply:
We should leave it as `training_loss`. You might change the loss in the future.

Reply:
Btw I removed loss in #2662